### ISSUE TITLE
.findElements are not working correctly with viewPager

### ISSUE THREAD

**Issue by twehlin** [selendroid/selendroid#983]
In our app we are using a viewPager where you can swipe between pages.
And we have some elements with the same id on both of these pages.
The problem is when you do a .findElements you get back a list with the id´s that are on the page that are not shown.
example:
----------------------------
| id     |     id 
|
|
----------------------------

**Comment by EvilOverlordX** [comment]
Are the different pages really pages, or are they just different fragments?  It sounds like they aren't real pages from the DOM's point of view.  You may want to try using XPath locators instead of bare IDs.

**Comment by twehlin** [comment]
Hey, no you are right they are fragments.
But still if they are not visible, ie out of screen should i still get true returned on .isDisplayed?

**Comment by EvilOverlordX** [comment]
Yes, selendroid will have isDisplayed() return true if an element is
offscreen.

**Comment by twehlin** [comment]
But in that case it's a bug.
isDisplayed should @return Whether or not the element is displayed
So if its not displayed we should NOT get a true back.

  /**
   * Is this element displayed or not? This method avoids the problem of having to parse an
   * element's "style" attribute.
   *
   * @return Whether or not the element is displayed
   */
  boolean isDisplayed();

**Comment by EvilOverlordX** [comment]
I agrew.

John.
